.github/CODEOWNERS

```diff
@@ -265,13 +265,13 @@ components/skus/common/skus_utils.h @brave/sec-team
 # for brave_exec_script_allowlist
 build/dotfile_settings.gni @brave/brave-core-owners
 
-docs/gni_sources.md @bridiver
-docs/keyed_services.md @bridiver
+docs/gni_sources.md @brave/docs-reviewer
```

Comment:
not all docs should have the same reviewers

Reply:
Not all docs need to be in the code owners file. Lmk what name you want here